res/res_pjsip_authenticator_digest.c

```diff
@@ -178,6 +178,7 @@ static pj_status_t digest_lookup(pj_pool_t *pool, const pj_str_t *realm,
 	case AST_SIP_AUTH_TYPE_MD5:
 		pj_strdup2(pool, &info->data, auth->md5_creds);
 		info->data_type = PJSIP_CRED_DATA_DIGEST;
+		info->algorithm_type = PJSIP_AUTH_ALGORITHM_MD5;
```

Comment:
Do this need to be guarded with `#ifdef`s for non-bundled builds?

Reply:
Technically yes however...

Before this PR gets merged, there will be an addition to third-party/pjproject/patches that fixes the default so this line won't be needed.  Anyone who runs non-bundled will find that 2.15 won't auth using pre-hashed passwords and will either need to wait for 2.15.1 or build from pjproject source.

I sent an email to pjproject explaining what happened (I messed up the defaults) and told them I'd have a PR for them tomorrow.  Let's wait to see what their plans are for the PR.  Who knows, they may simply do a 2.15.1 now.

Even if they don't, right after this PR merges, the SHA-256/SHA-256-512 PR will go in which provides a definition for PJSIP_AUTH_ALGORITHM_MD5 and has the proper ifdef's.